heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -148,63 +162,63 @@ function process_message()
     if type(did) ~= "string" then return 0 end
     msg.Fields.documentId.value = did
 
-    local country = read_message("Fields[geoCountry]") or "Other"
-    if country == "??" then country = "Other" end
-    msg.Fields.country.value = country
-
+    msg.Fields.country.value = fx.normalize_country(read_message("Fields[geoCountry]"))
     msg.Fields.channel.value = fx.normalize_channel(read_message("Fields[appUpdateChannel]"))
-
-    msg.Fields.os.value = fx.normalize_os(read_message("Fields[os]"))
-
-    msg.Fields.hours.value = get_hours()
-
+    msg.Fields.os.value      = fx.normalize_os(read_message("Fields[os]"))
     msg.Fields.default.value = is_default_browser()
 
-    local cnts = get_search_counts()
-    msg.Fields.google.value = cnts[1]
-    msg.Fields.bing.value   = cnts[2]
-    msg.Fields.yahoo.value  = cnts[3]
-    msg.Fields.other.value  = cnts[4]
-
-    msg.Fields.reason.value             = ""
-    msg.Fields.sessionId.value          = ""
-    msg.Fields.subsessionCounter.value  = 0
-    msg.Fields.buildId.value            = ""
-    msg.Fields.pluginHangs.value        = 0
-
-    -- add session information for broken session monitoring
+    msg.Fields.reason.value = ""
     local reason = read_message("Fields[reason]")
     if type(reason) == "string" then
         msg.Fields.reason.value = reason
+    elseif doc_type == "crash" then
+        msg.Fields.reason.value = "es.crash"
```

Comment:
I don't feel super happy about re-using the "reason" field, I'd prefer introducing another field for `docType` to distinguish crashes vs. main pings. If we're not planning to incorporate any other docTypes into the summary stream, I'm willing to punt on this for now.